data_safe_haven/resources/workspace/ansible/desired_state.yaml

```diff
@@ -67,14 +67,12 @@
             fi
           fi
 
-    # This will create directories if src or dest ends in '/'
```

Comment:
I think it's worth leaving a note to this effect somewhere - it's not totally obvious when this fails.

Reply:
We could leave one? The duplication bothers me.

It feels like an odd comment as it is expected behaviour. I think it is clear what the task is doing.

Reply:
It wasn't obvious to me why

```
src: etc/xrdp
dest: /etc/xrdp
```

was failing before, but I agree - this is standard behaviour. Happy to just drop it.

Reply:
It is definitely a quirk (and maybe a bad design). I feel a bit like it isn't out job to document that but it could be confusing for a developer.